firmware/baseband/proc_capture.cpp

```diff
@@ -23,19 +23,21 @@
 #include "proc_capture.hpp"
 
 #include "dsp_fir_taps.hpp"
-
 #include "event_m4.hpp"
-
 #include "utility.hpp"
 
 CaptureProcessor::CaptureProcessor() {
     decim_0.configure(taps_200k_decim_0.taps, 33554432);
     decim_1.configure(taps_200k_decim_1.taps, 131072);
 
     channel_spectrum.set_decimation_factor(1);
+    ready = true;
```

Comment:
This is the actual workaround. Prevents the execute method from doing any work until the ctor has completed.

Reply:
This is not the "fix" -- but that's going to be a little more invasive (change to the threading). So I'll do that in another PR.

Reply:
Can that method also be used to solve https://github.com/eried/portapack-mayhem/issues/753 ?

Reply:
For #753, we'd need to consider something like a shutdown message or something that would give the baseband time to wrap up before actually exiting. I'll take a look.